From 07a04ecfa739dcce969b8726aa58f3496e6ad0d7 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sat, 9 Feb 2019 23:57:55 +0100
Subject: [PATCH 6/6] qtbase: Do not set PKG_CONFIG_SYSROOT_DIR when -sysroot
 passed

We carefully manage PKG_CONFIG_LIBDIR and we have pkg-config
files outside of the -sysroot anyway. PKG_CONFIG_SYSROOT_DIR
is a blunt tool, prepended to each path in PKG_CONFIG_LIBDIR

Change-Id: Ife0170c98f97656bb6de5357145cb13a6e068838
---
 configure.pri | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/qtbase/configure.pri b/qtbase/configure.pri
index 131aa868c2..6597a268bf 100644
--- a/qtbase/configure.pri
+++ b/qtbase/configure.pri
@@ -366,8 +366,8 @@ defineTest(qtConfTest_detectPkgConfig) {
                 return(false)
             }
 
-            pkgConfigSysrootDir = $$sysroot
-            qtConfAddNote("PKG_CONFIG_SYSROOT_DIR automatically set to $$pkgConfigSysrootDir")
+            # pkgConfigSysrootDir = $$sysroot
+            qtConfAddNote("WARNING :: Qt upstream would've set PKG_CONFIG_SYSROOT_DIR automatically set to $$pkgConfigSysrootDir, we do not!")
         }
         $${1}.pkgConfigLibdir = $$pkgConfigLibdir
         export($${1}.pkgConfigLibdir)
-- 
2.20.1

